x/epochs/types/genesis.go

```diff
@@ -0,0 +1,15 @@
+package types
+
+// DefaultIndex is the default capability global index
+const DefaultIndex uint64 = 1
+
+// DefaultGenesis returns the default Capability genesis state
+func DefaultGenesis() *GenesisState {
+	return &GenesisState{}
+}
+
+// Validate performs basic genesis state validation returning an error upon any
+// failure.
+func (gs GenesisState) Validate() error {
```

Comment:
Validation won't be required for epoch number range, start time range and etc?

Reply:
Will add validation